{
  final int SEGMENT_SIZE=64;
  final int NUM_VALUES=64;
  try {
    test(Util.randomRecords(NUM_VALUES,SerializationTestTypeFactory.INT),SEGMENT_SIZE);
  }
 catch (  Exception e) {
    e.printStackTrace();
    Assert.fail("Test encountered an unexpected exception.");
  }
}
